.github/ISSUE_TEMPLATE/feature.yaml

```
@@ -1,12 +1,12 @@
-name: 🚀 Feature"
+name: "🚀 Feature"
 description: "Submit a proposal for a new feature"
 title: "🚀 Feature: "
 labels: [feature]
 body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out our bug report form 🙏
+        Thanks for taking the time to fill out feature request report form 🙏
```

```suggestion
        Thanks for taking the time to fill out the feature request form 🙏
```

This makes more sense than "fill out feature request report form", IMHO. Up to you though. 👍

---

That's what happens, when in hurry 😄